### PR TITLE
fix: container based deployments on native projects

### PR DESCRIPTION
**Pull request by elorzafe**
*Issue #, if available:*

*Description of changes:*
Native projects were missing to enable container based deployments

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by nikhname — APPROVED** [review]
LGTM!

File: packages/amplify-provider-awscloudformation/src/configuration-manager.ts [hunk 1]
```diff
@@ -82,13 +83,33 @@ export async function configure(context: $TSContext) {
...
+  await enableServerlessContainers(context);
+
   await newUserCheck(context);
   printInfo(context);
   await setProjectConfigAction(context);
   return await carryOutConfigAction(context);
 }
 
-function doesAwsConfigExists(context: $TSContext) {
+async function enableServerlessContainers(context) {
```
Comment (jhockett) [inline packages/amplify-provider-awscloudformation/src/configuration-manager.ts:94]: nit: add type to parameter 
```ts
async function enableServerlessContainers(context: $TSContext) {
```

File: packages/amplify-provider-awscloudformation/src/configuration-manager.ts [hunk 2]
```diff
@@ -82,13 +83,33 @@ export async function configure(context: $TSContext) {
...
+  });
+
+  if (!context.exeInfo.projectConfig[frontend]) {
+    context.exeInfo.projectConfig[frontend] = { config };
+  }
+
+  context.exeInfo.projectConfig[frontend].config = { ...config, ServerlessContainers };
+}
+
+function doesAwsConfigExists(context) {
```
Comment (jhockett) [inline packages/amplify-provider-awscloudformation/src/configuration-manager.ts:112]: nit: revert back to `function doesAwsConfigExists(context: $TSContext)`

File: packages/amplify-provider-awscloudformation/src/configuration-manager.ts [hunk 3]
```diff
@@ -22,6 +22,7 @@ import {
   removeProjectComfirmQuestion,
   updateOrRemoveQuestion,
 } from './question-flows/configuration-questions';
+import inquirer from 'inquirer';
```
Comment (jhockett) [inline packages/amplify-provider-awscloudformation/src/configuration-manager.ts:25]: `inquirer` is already imported on line 7

**Review by jhockett — APPROVED** [review]

**Pull request merged**